leaf-ui/js/object/Class.js

```diff
@@ -802,8 +837,9 @@ class EVO {
      * Switch back to modeling slider, if EVO is on the visualization returns to filling by initial state.
      */
      static switchToModelingMode() {
-            $('#modelingSlider').css("display", "");
-            $('#analysisSlider').css("display", "none");
+        $('#modelingSlider').css("display", "");
```

Comment:
Generally I don't think we want css updates in our class method but since it seems like it was already there and the change is more of a github being sensitive thing its probably fine - but just wanted to document that since it is shown as a change in this pull request.